dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -303,4 +314,11 @@ public int compare(String s1, String s2) {
 
   }
 
+  private static ImmutableList<Node> toList(Nodes nodes) {
```

Comment:
This will no longer be necessary if I can ever convince Sonatype to let me upload XOM 1.3.0 to Maven Central (but I've failed twice so it may not be happening anytime soon)

Reply:
Looking forward.